jme3-core/src/main/resources/Common/MatDefs/ShaderNodes/Environment/envMapping100.frag

```diff
@@ -4,5 +4,9 @@ void main(){
     //@input samplerCube cubeMap the cube map
     //@output vec4 color the output color
 
-    color = textureCubeLod(cubeMap, refVec, 0.0);
-}
\ No newline at end of file
+    #ifdef GL_ES
+        color = textureCube(cubeMap, refVec);
```

Comment:
What is the reason for this?

Reply:
If using textureCubeLod the shader failed to compile because as stated in gles sl 100 specs "The built-ins suffixed with “Lod” are allowed only in a vertex shader" (page 71)

Reply:
Are we talking about GL ES 2.0 ? Because that's the minimum supported version in jme.
If that's the case, this extension should enable textureCubeLod https://www.khronos.org/registry/OpenGL/extensions/EXT/EXT_shader_texture_lod.txt

Reply:
Yes, I had a look at that extension but if the device don't support it, it'll fail, so I consider it safer to avoid using lod in glsl100

Reply:
The problem is that some env maps might not render properly because mipmaps levels could be used to store different informations, see for example pbr env maps.

Reply:
Yes, i think this is the best solution.

Reply:
Just comited